packages/opentelemetry-core/scripts/version-update.js

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const license =
+`/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+`;
+
+const fs = require('fs');
+const path = require('path');
+
+const appRoot = path.resolve(__dirname);
+const fileUrl = path.resolve(`${appRoot}/../src/common/version.ts`);
+const packageJsonUrl = path.resolve(`${appRoot}/../package.json`);
+const pjson = require(packageJsonUrl);
+const content = `
+// this is autogenerated file, see scripts/version-update.js
+export const VERSION: string = '${pjson.version};';
```

Comment:
Minor nit: I believe the `: string` is unncessary since the type is trivially inferred from the assigned string constant.